seed/models/building_file.py

```diff
@@ -278,18 +289,23 @@ def process(self, organization_id, cycle, property_view=None):
                 meter.source = m.get('source')
                 meter.type = m.get('type')
                 meter.is_virtual = m.get('is_virtual')
+                if meter.is_virtual is None:
+                    meter.is_virtual = False
                 meter.save()
 
                 # meterreadings
-                # TODO: need to check that these are in kBtu already?
+                # ENERGY_TYPES index starts at 1, tuple indexes start at 0
+                meter_type = Meter.ENERGY_TYPES[meter.type - 1][1]
+                meter_conversions = self._kbtu_thermal_conversion_factors().get(meter_type, {})
```

Comment:
Hmm, maybe we shouldn't assume enums match indices here. Maybe we could do something like this?
```
dict(Meter.ENERGY_TYPES)[meter.type]
```
Note that dict(<list of tuples>) will create a dict where the keys are the first values, and the values are the second values

Reply:
I like it, thanks!